integration-tests/smoke/vrfv3_test.go

```diff
@@ -0,0 +1,336 @@
+		LinkEthFeedAddress:    mockETHLinkFeed.Address(),
+	}
+	actions.CreateOCR2VRFV3Jobs(
+		t,
+		bootstrapNode,
+		nonBootstrapNodes,
+		ocr2VRFPluginConfig,
+		testNetwork.ChainID,
+		0,
+	)
+
+	ocr2DkgConfig := actions.BuildOCR2DKGConfigVars(t, ocr2VRFPluginConfig)
+
+	//13. set config for DKG OCR
+	log.Debug().Interface("OCR2 DKG Config: ", ocr2DkgConfig).Msg("OCR2 DKG Config")
+	err = dkg.SetConfig(
+		ocr2DkgConfig.Signers,
+		ocr2DkgConfig.Transmitters,
+		ocr2DkgConfig.F,
+		ocr2DkgConfig.OnchainConfig,
+		ocr2DkgConfig.OffchainConfigVersion,
+		ocr2DkgConfig.OffchainConfig,
+	)
+	require.NoError(t, err)
+
+	//14. wait for the event ConfigSet from DKG contract
+	dkgConfigSetEvent, err := dkg.WaitForConfigSetEvent()
+	log.Info().Interface("Event: ", dkgConfigSetEvent).Msg("OCR2 DKG Config was set")
+	//15. wait for the event Transmitted from DKG contract, meaning that OCR committee has sent out the Public key and Shares
+	dkgSharesTransmittedEvent, err := dkg.WaitForTransmittedEvent()
+	log.Info().Interface("Event: ", dkgSharesTransmittedEvent).Msg("DKG Shares were generated and transmitted by OCR Committee")
+
+	ocr2VrfConfig := actions.BuildOCR2VRFConfigVars(t, ocr2VRFPluginConfig)
+	log.Debug().Interface("OCR2 VRF Config: ", ocr2VrfConfig).Msg("OCR2 VRF Config")
+
+	//16. set config for VRFBeacon OCR
+	err = vrfBeacon.SetConfig(
+		ocr2VrfConfig.Signers,
+		ocr2VrfConfig.Transmitters,
+		ocr2VrfConfig.F,
+		ocr2VrfConfig.OnchainConfig,
+		ocr2VrfConfig.OffchainConfigVersion,
+		ocr2VrfConfig.OffchainConfig,
+	)
+	require.NoError(t, err)
+
+	//15. wait for the event ConfigSet from VRFBeacon contract
+	vrfConfigSetEvent, err := vrfBeacon.WaitForConfigSetEvent()
+	log.Info().Interface("Event: ", vrfConfigSetEvent).Msg("OCR2 VRF Config was set")
+
+	// TODO - currently there VRF OCR fails, need to investigate "latest vrf ConfigSet by sig with confs" error
+
+	receipt, err := consumer.RequestRandomness(
+		2,
+		subId,
+		big.NewInt(1),
+	)
+
+	require.NoError(t, err)
+
+	//IBeaconPeriodBlocks is also stored in consumer contract
+	periodBlocks, err := consumer.IBeaconPeriodBlocks(nil)
+	require.NoError(t, err)
+
+	blockNumber := receipt.BlockNumber
+	periodOffset := new(big.Int).Mod(blockNumber, periodBlocks)
+	nextBeaconOutputHeight := new(big.Int).Sub(new(big.Int).Add(blockNumber, periodBlocks), periodOffset)
+
+	confDelay := big.NewInt(1)
+	requestID, err := consumer.GetRequestIdsBy(nil, nextBeaconOutputHeight, confDelay)
+	require.NoError(t, err)
+
+	//todo - Wait until OCR committee will publish randomness - Check NewTransmission event in VRFBeacon
+	err = consumer.RedeemRandomness(requestID)
+	require.NoError(t, err)
+
+	randomness, err := consumer.GetRandomnessByRequestId(nil, requestID, big.NewInt(0))
+	require.NoError(t, err)
+	log.Info().Interface("Random Number: ", randomness).Msg("Randomness generated")
+
+}
+
+func setupVRFv3Test(t *testing.T) (testEnvironment *environment.Environment, testNetwork blockchain.EVMNetwork) {
```

Comment:
```suggestion
func setupOCR2VRFTest(t *testing.T) (testEnvironment *environment.Environment, testNetwork blockchain.EVMNetwork) {
```

Reply:
asked similar question, waiting for reply there

Reply:
renamed